tko_account_pt/__openerp__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Thinkopen - Portugal & Brasil
+#    Copyright (C) Thinkopen Solutions (<http://www.thinkopensolutions.com>).
+#
+#    $Id$
+#
+#    This module was developed by ThinkOpen Solutions for OpenERP as a
+#    contribution to the community.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version $revnoof the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Contabilidade PT',
+    'version': '1.122',
+    'author': 'ThinkOpen Solutions,Odoo Community Association (OCA)',
+    'license': 'Other OSI approved licence',
```

Comment:
Please specify the right license, as far as I see in you headers, it is GPL and GPL v3 or are you planning to set it as LGPL waiting for the key to be valid https://github.com/odoo/odoo/pull/6914?